stdlib/public/core/Integers.swift.gyb

```diff
@@ -1918,6 +1924,18 @@ extension BinaryInteger {
 }
 #endif
 
+extension BinaryInteger {
+  // FIXME(integers): inefficient. Should get rid of _word(at:) and
+  // countRepresentedWords, and make `words` the basic operation.
```

Comment:
I was thinking about `CollectionOfOne<UInt>`. But `_UIntBuffer<UInt, UInt64>` looks like a better solution for 32-bit platforms. Thanks!